hewr-vazhin/style.css

```diff
@@ -0,0 +1,487 @@
+  flex-direction: row;
+  justify-content: center;
+  padding: 20px;
+  width: 100%;
+}
+
+#flex-challenge {
+  width: 80%;
+  margin: 20px 20px 5px 20px;
+  box-shadow: 1px 1px 1px 1px gray;
+  display: flex;
+  justify-content: center;
+}
+
+#section-4 #a-b-c-d-e {
+  display: flex;
+  flex-direction: column;
+}
+
+#f-g-h {
+  display: flex;
+}
+
+#section-4 #a-b {
+  display: flex;
+  flex-direction: row;
+}
+
+#section-4 #a {
+  flex-basis: 35%;
+}
+
+#section-4 #b {
+  flex-basis: 65%;
+}
+
+#section-4 #c-d-e {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+}
+
+#section-4 #c-d {
+  display: flex;
+  flex-direction: column;
+}
+
+#section-4 #e {
+  width: 30rem;
+  height: 94%;
+}
+
+#section-4 #g-h {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+}
+
+#section-4 #g {
+  height: 80%;
+}
+
+#section-4 #h {
+  height: 20%;
+}
+
+#section-4 #f {
+  height: 96%;
+}
+
+#section-4 .grey-box {
+  animation: color-animation 5s ease-in-out;
+  animation-iteration-count: infinite;
+  width: 10rem;
+  height: 10rem;
+  margin: 10px;
+}
+
+#section-4 .grey-box p {
+  font-size: 1.2rem;
+  color: white;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+#section-5 {
+  display: flex;
+  flex-direction: column;
+  background: #fafafa;
+  padding: 1rem;
+  color: #808080;
+  padding-bottom: 80px;
+}
+
+#section-5 div p {
+  text-align: justify;
+  font-size: 0.65rem;
+  width: 300px;
+  margin: 2px;
+}
+
+#section-6 {
+  height: 80vh;
+  background-color: #656565;
+}
+
+#section-6 section h2 {
+  color: white;
+  font-size: 2.5rem;
+  font-family: 'Reem Kufi', sans-serif;
+}
+
+#experience-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+#experience-container div {
+  margin-top: .8rem;
+  width: 70%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.job-company {
+  display: flex;
+  flex-direction: column !important;
+}
+
+#section-6 {
+  flex-direction: column;
+}
+
+#footer-sec-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 400px;
+}
+
+#footer-sec-1 form {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: space-evenly;
+}
+
+#footer-sec-1 form textarea,
+#footer-sec-1 form .input {
+  padding: 5px 0;
+  text-align: center;
+  font-family: 'Inconsolata', monospace;
+  font-size: 1.1rem;
+}
+
+#footer-sec-1 form textarea {
+  resize: none;
+  text-align: left;
+  padding-left: 8px;
+}
+
+.btn {
+  font-size: 1.4rem;
+  width: 50%;
+  margin: 0 auto;
+  padding: .5rem 1rem;
+  background: white;
+  cursor: pointer;
+  border: none;
+  color: #000;
+  transition: transform ease .1s;
+  border-radius: 5px;
+  font-family: 'Inconsolata', monospace;
+}
+
+.btn:hover {
+  /* border: .5px solid  rgb(165, 192, 192);
+  padding: 10px;
+  box-shadow: 1px 1px 1px 1px black; */
```

Comment:
try not to leave random comment lines in your code, if you don't have to them delete them before you made the PR